main.go

```diff
@@ -35,6 +36,12 @@ func main() {
 	// log startup message
 	log.Printf("Starting estafette-extension-npm-audit version %v...", version)
 
+	var levelString string
+	if *level != "" {
```

Comment:
You can set a default in kingpin.Flag with 
```
level = kingpin.Flag("level", "...").Default("low").OverrideDefaultFromEnvar("ESTAFETTE_EXTENSION_LEVEL").String()
```

This way you remove this code entirely and use `*level` below.